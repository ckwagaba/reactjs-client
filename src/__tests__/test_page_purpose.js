import React from 'react';
import { shallow } from 'enzyme';
import PagePurpose from '../components/page_purpose';

describe('PagePurpose component', () => {

  const component = <PagePurpose />;

  it('should exist', () => {
    expect(component).toMatchSnapshot();
  });

});
