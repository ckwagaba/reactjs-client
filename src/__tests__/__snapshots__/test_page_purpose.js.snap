// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PagePurpose component should exist 1`] = `<PagePurpose />`;
